articles/devops-project/azure-devops-project-iot.md

```diff
@@ -0,0 +1,138 @@
+# Continuously deploy to IoT Edge applications with DevOps Projects
+
+Azure DevOps Projects presents a simplified experience where you can bring your existing code and Git repo or choose a sample application to create a continuous integration (CI) and continuous delivery (CD) pipeline to Azure.
+
+DevOps Projects also:
+
+- Automatically creates Azure resources, such as IoTHub
+
+- Creates and configures a release pipeline in Azure DevOps that sets up a build and release pipeline for CI/CD
+
+In this tutorial, you will:
+
+- Use DevOps Projects to deploy an ASP.NET app to IoT Edge application

```

Comment:
No asp.net. Developers can deploy .netcore, node.js, java, c and python modules.